### PR TITLE
Enable code coverage for codecov

### PR DESCRIPTION
**Pull request by davidbeckingsale**

### CODE REVIEW

**Comment by davidbeckingsale** [comment]
@mcfadden8 - our AllocationMap "performance" test is failing. Do you think we should just get rid of it, or skip it for Travis builds?

**Comment by mcfadden8** [comment]
Why is the performance test failing?  I'd like to understand this better and understand whether there are viable alternatives.

File: codecov.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ignore:
+  - "src/umpire/tpl"
```
Comment (mcfadden8) [inline codecov.yml]: Won't this also ignore simPool?  I think we want to add that right?

File: tests/unit/util/CMakeLists.txt [hunk 2]
```diff
@@ -32,12 +32,15 @@ blt_add_test(
   NAME allocation_map_tests
   COMMAND allocation_map_tests)
 
-blt_add_executable(
-  NAME allocation_map_performance_tests
-  SOURCES allocation_map_performance_tests.cpp
-  DEPENDS_ON umpire_util gtest
-  OUTPUT_DIR ${UMPIRE_TEST_OUTPUT_DIR})
+# TODO: Make this test a relative performance test
```
Comment (mcfadden8) [inline tests/unit/util/CMakeLists.txt]: Can we only disable this when running coverage tests?

Comment (davidbeckingsale) [reply tests/unit/util/CMakeLists.txt]: That was previously done - it didn't help.

File: codecov.yml (hunk 1 above)
Comment (davidbeckingsale) [reply codecov.yml]: Yes. I can change that.

**Comment by davidbeckingsale** [comment]
👍

**Pull request merged**